lib/Authen/SASL.pod

```diff
@@ -79,6 +79,14 @@ See the L<callback|/callback> method for details.
 Set the list of mechanisms to choose from.
 See the L<mechanism|/mechanism> method for details.
 
+=item maxssf =E<gt> VALUE
+
+Override the SASL Maximum Security Strength Factor to C<VALUE>. This is required
+to be 0 for non-standard server implementations (ie. Active Directory) that do not 
+support SASL-layer encryption or integrity on SSL/TLS-protected connections.
+
+The default value is 2**31 - 1 for the Perl module, and 255 for the XS module.
```

Comment:
@ngarratt where can I see this default? I'm unable to find where this default is being set or coming from. Is that coming from `Net::LDAP`? I think it's good to change the documentation to

```suggestion
The default selection is delegated to L<Net::LDAP> for the Perl module (which uses is 2**31 - 1), and 255 for the XS module.
```

Reply:
@ngarratt any comments to my suggested change?